### ISSUE TITLE
Some elements can't be navigated on Edge and IE when use "Tab" key

### ISSUE THREAD

**Issue by Helen09** [Microsoft/pxt-adafruit#642]
**Repro steps:**
• Navigate to https://makecode.adafruit.com/beta
• Use "tab" to navigate every element on this page

**Except result: (Chrome & Firefox)**
Every element should be navigated on this page
![3](https://user-images.githubusercontent.com/23466737/36584506-dee911b2-1871-11e8-9c89-fa3eb71b0031.gif)

**Actual result: (Edge & IE)**
There is a trap on IE when navigated to "help" and some elements can't be navigated on Edge
![2](https://user-images.githubusercontent.com/23466737/36584504-d9b14e9e-1871-11e8-8b08-b7558daa1825.gif)

**Comment by samelhusseini** [comment]
Most likely fixed by nojquery change as that changes how we fundamentally do focus.

**Comment by samelhusseini** [comment]
Fixed in /beta please verify and re-open if not.

**Comment by Helen09** [comment]
I have verified this bug and it has been fixed

**Comment by samelhusseini** [comment]
Thank you
